bin/bb8/docker_rsync.py

```diff
@@ -28,7 +28,6 @@ def _run_rsync(self, volumes, from_path, to_path, relative):
                # to the default rsh)
                "-e", ssh_cmd,
                "--perms", "--owner", "--group",
```

Comment:
Can we also drop the owner and group flags too?

Reply:
Wait, do we even need the `--perms` flag in the end?

Reply:
oh yeah nevermind, I'm getting mixed up